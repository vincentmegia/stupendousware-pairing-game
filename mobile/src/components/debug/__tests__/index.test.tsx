import React from 'react'
import {render} from '@testing-library/react-native'
import Debug from '..'

describe('home screen tests', () => {
  it('it matches snapshots', () => {
    const {toJSON} = render(<Debug />)
    expect(toJSON()).toMatchSnapshot()
  })
})
