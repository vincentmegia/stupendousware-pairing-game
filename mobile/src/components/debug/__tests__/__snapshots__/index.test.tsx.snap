// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home screen tests it matches snapshots 1`] = `
<Text
  collapsable={false}
  style={
    Object {
      "color": "grey",
      "fontSize": 10,
      "marginTop": 80,
    }
  }
>
  0
</Text>
`;
